source/gwork/include/Gwork/Controls/CollapsibleList.h

```diff
@@ -61,10 +61,10 @@ namespace Gwk
             {
                 Base::List& children = GetChildren();
 
-                for (Base::List::iterator iter = children.begin(); iter != children.end(); ++iter)
+                for (auto&& iter : children)
```

Comment:
Good use of `auto&&`. I keep forgetting about this. I swap between languages a lot!

This is no longer an iterator though, you have unwrapped it by using the for-range, so it is now a child, e.g.
```cpp
for (auto&& child : children) { ... }
```

I know this sounds pedantic, but the point of using the for-range is to make the code easier to read/understand, so the name is important.